tests/python/gaia-ui-tests/gaiatest/tests/brick/test_brick_verification.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def setUp(self):
+        # Open Firefox OS and Skip the First Time Use app
+        GaiaTestCase.setUp(self)
+        # Open the homescreen app
+        self.homescreen = Homescreen(self.marionette)
+
+    def test_wait_for_homescreen(self):
+        self.apps.switch_to_displayed_app()
+        self.homescreen.wait_for_homescreen_visible()
```

Comment:
The element your waiting for here isn't really one that's interactable. I would suggest asserting that the displayed app is the homescreen (although I believe that's covered by a unit test), or perhaps waiting for at least one app icon to be displayed.